features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,200 @@
+---
+description: >-
+  A comprehensive guide that covers how you can add images and text over
+  a base image.
+---
+
+# Overlay using layers (beta)
+
+{% hint style="info" %}
+Overlay on images using layers is currently in beta.
+{% endhint %}
+
+In ImageKit, you can add images and text over a base image using [layers](#layers). Skip to the relevant section to understand with a quick example:
+
+* [Add images over image](overlay-using-layers#add-images-over-image)
+* [Add text over image](overlay-using-layers#add-text-over-image)
+* [Add solid color blocks over image](overlay-using-layers#add-images-over-image)
+
+# Layers
+A layer is a special kind of transformation that allows you to modify the overlay itself and express its position in relation to the parent.
```

Comment:
I think to make this text more complete, it should be something like "a layer transformation allows you to specify an asset to be used as an overlay, along with its positioning and transformations"

Reply:
How does this sound?
> A layer is a special kind of transformation in which you can specify an asset to be used as an overlay, along with its positioning and transformations. It supports nesting, allows you to modify the overlay itself, and express its position in relation to the parent.

Reply:
looks good

Reply:
Done